### DISCUSSION TITLE
SD card dump index counts reset

### DISCUSSION THREAD

**Discussion by pizzzza19** [sanni/cartreader#1047] [Need help]
system automatically count the dump number for SD card index orders.
Sometimes i feel reset SD index number but format does not reset it.
I think PCB rom remembers the counts. How to reset it?

**Comment by nsx0r** [comment]
go to the "About" menu to display the software version then press the button for 5 seconds (the LED colour will change), then release the button and you'll see the message "resetting folders..." and the OSCR will reboot.

**Reply by pizzzza19** [reply]
Thanks and it works as described!
